Add --protocol options for alignment with different data types (minimap2) (#23)
nf-core/nanoseq: docs/usage.md

## Alignment

### `--protocol`

> Specifies the data type being inputted (DNA, cDNA or directRNA)

Reviewer: ```suggestion
Specifies the type of data that was sequenced i.e. "DNA", "cDNA" or "directRNA".
```

### `--stranded`

> Specifies whether the inputted fastq files contain stranded data (default: false)

Reviewer: ```suggestion
Specifies if the data is strand-specific. Automatically activated when using --protocol directRNA (default: false)
```

> The specific command line arguments for `graphmap` and `minimap2` using `--protocol` and `--stranded` are specified below:

Reviewer: ```suggestion
When using `--protocol`/`--stranded` the following command-line arguments will be set for `minimap2` and `graphmap`:
```

| `nanoseq` input              | `minimap2` presets  | `graphmap` presets |
|------------------------------|---------------------|--------------------|
| `--protocol DNA`             | -ax map-ont         | tba                |
| `--protocol cDNA`            | -ax splice          | tba                |
| `--protocol directRNA`       | -ax splice -uf -k14 | tba                |
| `--protocol cDNA --stranded` | -ax splice -uf      | tba                |

> 

Reviewer: ```suggestion
```

Reviewer: Removing this line should get the markdownlint tests passing. Pipeline tests are purring 😃
